src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
What about equals and hashcode? Same for Product but do not forget to exclude lazy fields from it